src/EDMF_Environment.jl

```diff
@@ -59,6 +59,15 @@ function microphysics(
         if edmf.moisture_model isa NonEquilibriumMoisture
             aux_en.ql_tendency_precip_formation[k] = mph.ql_tendency * aux_en.area[k]
             aux_en.qi_tendency_precip_formation[k] = mph.qi_tendency * aux_en.area[k]
+
+            # keep track of non_eq_microphysics tendency part....
+            T_en = aux_en.T[k]
+            q_en = TD.PhasePartition(aux_en.q_tot[k], aux_en.q_liq[k], aux_en.q_ice[k])
+            ts_en= TD.PhaseNonEquil_pTq(param_set, p_c[k], T_en, q_en)
```

Comment:
```suggestion
            ts_en = ts_env[k]
```
I think that this should work too, since we already compute the environmental thermo state in `update_aux.jl` before this is called.

Also, above in this function, we'll need to add `ts_env = center_aux_environment(state).ts`.